ghost/core/core/server/services/members/service.js

```diff
@@ -6,6 +6,7 @@ const db = require('../../data/db');
 const MembersConfigProvider = require('./config');
 const MembersCSVImporter = require('@tryghost/members-importer');
 const MembersStats = require('./stats/members-stats');
+const memberJobs = require('./jobs/jobs');
```

Comment:
I think leaving the pattern that exists in the `analytics` jobs (index.js file in the root) would be best for consistency sake. It should not be full of logic though :D

Reply:
I wanted to keep away from overloading `index.js` file as we generally avoid them, no strong prefs tho so happy to change it back to index here :)

Reply:
Updated back to `index.js` ✔️